capstone/src/main/webapp/index.html

```diff
@@ -11,5 +13,7 @@ <h2>Welcome to UniViz!</h2>
       <input type="text" id="search" name="search"><br>
       <input type="submit" value="See your colleges!">
     </form>
+    <div id="calendar-freq-map"></div>
+    <div id="timeline-chart"></div>
```

Comment:
ive merged the ```dashboard.html``` file, where you can start adding these charts in the ```data``` div. you can rename it whatever you want, but i would suggest having your own ```data``` div where you have all the divs for the different charts.

Reply:
Ok I'll merge the master branch into this branch and add the charts to the dashboard.html file.

Reply:
On that same note, can we use a more specific div id now that we know what is going inside it? `data` is too generic.